WordPress/Classes/Services/PostService.m

```diff
@@ -734,6 +734,9 @@ - (RemotePost *)remotePostWithPost:(AbstractPost *)post
     remotePost.authorAvatarURL = post.authorAvatarURL;
     remotePost.excerpt = post.mt_excerpt;
     remotePost.slug = post.wp_slug;
+    if(post.permaLink != nil) {
```

Comment:
I think the change in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/163 renders this change moot since we're now fetching the canonical remote post.  Probably fine to keep as is, just making a note for future reference.

Reply:
@aerych , I left this change since we are returning the captured post in the case of failure to fetch and I wanted it to still have the url value